cfgov/unprocessed/apps/analytics-gtm/webpack-config.js

```diff
@@ -37,14 +38,19 @@ const COMMON_MODULE_CONFIG = {
         presets: [ [ 'babel-preset-env', {
           targets: {
             browsers: BROWSER_LIST.LAST_2_IE_9_UP
-          },
-          debug: true
```

Comment:
I'd be inclined to leave this as `debug: false` so it's easier to find where to turn it on, like is done with the uglify config. Or do you think that's too much clutter?

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/3970/commits/cc2efa06e6dbd3b47a7078f290cf89ead46dffa4.